packages/commonwealth/client/scripts/controllers/server/threads.ts

```diff
@@ -174,7 +174,9 @@ class ThreadsController {
     const chainEntitiesProcessed: ChainEntity[] = [];
     if (chain_entity_meta) {
       for (const meta of chain_entity_meta) {
-        const full_entity = app.chainEntities.store.getById(meta.ce_id);
+        const full_entity = Array.from(app.chainEntities.store.values())
```

Comment:
Can we encapsulate all these calls to the store behind meaningfully named methods on the `app.chainEntities` controller? Generally we shouldn't be accessing the store directly in modules outside of the controller itself.